.idea/runConfigurations.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this should go in `.gitignore`
You can use a glob pattern to ignore this kind of files